constants.rb

```diff
@@ -0,0 +1,8 @@
+module Constants
+  BIKE_MAX_DISTANCE = 30
+  BIKE_MAX_WEIGHT = 10
+  BIKE_MAX_SPEED = 10
+
+  CAR_MAX_WEIGHT = 100
+  CAR_MAX_SPEED = 50
+end
```

Comment:
Normally, in the end of all files should be a newline. Consider adding this here.

Reply:
Done.